File: app/src/main/java/com/jeketos/associatewith/Point.java
package com.jeketos.associatewith;

public class Point {
    public float x;
    public float y;
    public int motionEvent;

    public Point(float x, float y, int motionEvent) {
        this.x = x;
        this.y = y;
        this.motionEvent = motionEvent;
    }

    @Override
    public String toString() {
        return "x = " + x +
                "\ny = " + y +
                "\nevent = " + motionEvent;
    }
}

File: app/src/main/java/com/jeketos/associatewith/Point.kt
package com.jeketos.associatewith

/**
 * Created by jeketos on 05.12.2016.
 *
 */
class Point(var x : Float, var y : Float, var motionEvent : Int, var color : Int, var strokeWidth : Float) {

     override fun toString(): String {
        return "x = " + x +
                "\ny = " + y +
                "\nevent = " + motionEvent +
                "\ncolor = " + color +
                "\nstrokeWidth = " + strokeWidth
    }
}